src/app/utils/socket-utils.ts

```diff
@@ -377,7 +405,7 @@ export function getGeneralSockets(
     c.category.uiCategoryStyle !== ARMOR_STAT_CATEGORYSTYLE;
 
   const isAllowedSocket = (socketInfo: DimSocket) =>
-    socketInfo.socketIndex !== intrinsicSocket?.socketIndex &&
+    !intrinsicSockets?.find((s) => s.socketIndex === socketInfo.socketIndex) &&
```

Comment:
Using 'every' or otherwise here causes this check to fail if intrinsicSockets is undefined, which is why this is implemented like this. If anyone knows a better way to clean it up, please let me know.

Reply:
Could also make intrinsicSockets non-nullable, but unsure whether that's the play.